docs/user/dql/functions.rst

```diff
@@ -2769,6 +2769,46 @@ Another example to show how to set custom values for the optional parameters::
     +----------------------+--------------------------+
 
 
+MATCHPHRASEQUERY
+------------
+
+Description
+>>>>>>>>>>>
+
+``matchphrasequery(field_expression, query_expression[, option=<option_value>]*)``
+
+The matchphrasequery function maps to the match_phrase query used in search engine, to return the documents that match a provided text with a given field.
+It is an alternate syntax for the `match_phrase`_ function. Available parameters include:
+
+- analyzer
+- slop
+- zero_terms_query
+
+For backward compatibility, matchphrase is also supported and mapped to match_phrase query as well.
```

Comment:
```suggestion
For backward compatibility, matchphrase is also supported and mapped to match_phrase query.
```

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/165/commits/d04916bfbe558347bec18a9b61b3f13697ce270d